spec/models/dynamic_field_spec.rb

```diff
@@ -0,0 +1,186 @@
+require 'rails_helper'
+
+RSpec.describe DynamicField, type: :model do
+  describe '#new' do
+    context 'when parameters are correct' do
+      subject { FactoryBot.create(:dynamic_field) }
```

Comment:
Just as a heads up, rubocop will be complaining about not having a "named subject" here once my bixby changes are merged in.  I'm fine with fixing this later, but it might be faster to fix now while this set of tests is fresh in your mind.

https://www.rubydoc.info/gems/rubocop-rspec/1.6.0/RuboCop/Cop/RSpec/NamedSubject

Reply:
Can you push the bixby changes to 3.x? There's going to be a lot more errors than just this one.

Reply:
Will do.

Reply:
Thanks let me know!